packages/dropdowns/stories/examples/Menu/Default.tsx

```diff
@@ -30,21 +30,23 @@ interface IStoryProps {
   isAnimated: boolean;
   isCompact: boolean;
   isDisabled: boolean;
+  isOpen: boolean;
 }
 
 export const Default: Story<IStoryProps> = ({
   hasArrow,
   isAnimated,
   isCompact,
   placement,
-  isDisabled
+  isDisabled,
+  isOpen
 }) => {
   return (
     <Grid>
       <Row style={{ minHeight: 200 }}>
         <Col textAlign="center">
           {/* eslint-disable-next-line no-alert */}
-          <Dropdown onSelect={item => alert(item)}>
+          <Dropdown onSelect={item => alert(item)} isOpen={isOpen || undefined}>
```

Comment:
What do you think about making this example a fully controlled component? I think it would get rid of the warnings in the console. Some [other](https://github.com/reach/reach-ui/blob/develop/packages/accordion/src/index.tsx#L77) [libraries](https://github.com/facebook/react/blob/master/packages/react-dom/src/client/ReactDOMInput.js#L39) also log a warning for switching usages during the component life cycle.

<img src='https://user-images.githubusercontent.com/1811365/117216709-f3be8500-adb4-11eb-9b7f-e679555da6d3.gif' alt='screenshot' width='500' />

Reply:
ack, yeah I'll get these updated as you've suggested.

Reply:
leaving as-is for now per offline conversation.